pkg/tbtc/dkg.go

```diff
@@ -139,6 +136,23 @@ func (drl *dkgRetryLoop) start(
 			}
 		}
 
+		// Wait for the right moment to execute the attemptFn, as calculated
+		// in drl.attemptStartBlock.
+		err := waitForDkgAttemptFn(ctx, drl.attemptStartBlock)
+		if err != nil {
+			return nil, 0, fmt.Errorf(
+				"failed waiting on block [%v] for attempt [%v]: [%v]",
+				drl.attemptStartBlock,
+				drl.attemptCounter+1,
```

Comment:
Why do we increment it? Shouldn't it match the value we pass to the `dkgAttemptFn` in line 162? It's already incremented in the `for` loop beginning.

Reply:
Yes, this is wrong. Fixed in 749fd2a3cd0293fb605054a8d5c9010ba4948545.